JavaAssignment/src/manasiM/Swap_numbers.java

```diff
@@ -0,0 +1,22 @@
+package manasiM;
+
+public class Swap_numbers 
+{
+	void swap(int x, int y)
+	{
+		System.out.println("Before Swap X is " +x +" and Y is "+y);
+		int temp=0;
```

Comment:
No need to assign, local variable with 0, instead of directly write like "int temp= x; "